Expensify/App
# Add terms to signin/settings pages

<If necessary, assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit.>

### Details
<Explanation of the change or anything fishy that is going on>
1. Adds terms of service and privacy policy links to settings modal. 
2. Adds terms of service and privacy policy links (only) on mobile apps/desktop
3. Adds terms of service, privacy policy, and licenses links on web/mobile web

### Fixed Issues
<Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing>
Fixes https://github.com/Expensify/Expensify/issues/154672

### Tests

1. Go to the sign in page on web & mobile web
2. Make sure you see terms of service, privacy policy, and licenses links and that they work
3. Go to the sign in page on ios/android/desktop
4. Make sure you see only the terms of service and privacy policy links and that they work
5. Sign in on each platform and tap your avatar to launch the settings modal 
6. Make sure you see the terms/privacy policy links at the bottom of the modal and that they work

### Tested On


### Screenshots


#### Web

![image](https://user-images.githubusercontent.com/1371996/111941782-d2713600-8a9f-11eb-82a1-6e5471155c06.png)
![image](https://user-images.githubusercontent.com/1371996/111943608-bcfe0b00-8aa3-11eb-914a-0d403c740f5d.png)

#### Mobile Web

![image](https://user-images.githubusercontent.com/1371996/111942434-43fdb400-8aa1-11eb-9ec9-3f0a8edeb515.png)
![image](https://user-images.githubusercontent.com/1371996/111943744-11a18600-8aa4-11eb-8fbf-4e11a116a84e.png)

#### Desktop

![image](https://user-images.githubusercontent.com/1371996/111942623-9fc83d00-8aa1-11eb-9b63-984f34455dbd.png)
![image](https://user-images.githubusercontent.com/1371996/111943823-47df0580-8aa4-11eb-99c7-b9453cc7de22.png)

#### iOS

![image](https://user-images.githubusercontent.com/1371996/111942272-fe40eb80-8aa0-11eb-8a27-32b9c81e5f32.png)
![image](https://user-images.githubusercontent.com/1371996/111943681-e9b22280-8aa3-11eb-83ff-117841aa6e2d.png)

#### Android

![image](https://user-images.githubusercontent.com/1371996/111941933-37c52700-8aa0-11eb-914d-5f6468b8262a.png)
![image](https://user-images.githubusercontent.com/1371996/111943921-7ceb5800-8aa4-11eb-8f4f-bdf48e4f1568.png)
